server/server.ts

```diff
@@ -65,12 +70,14 @@ app.use(morgan('outFormat', {skip: (req : express.Request, res : express.Respons
 app.use('/', root);
 app.use('/posts', postRoutes);
 app.use('/users', userRoutes);
-app.use('/posts/:id/comments', replyRoutes);
+app.use('/comments', replyRoutes);
+app.use('/like', likeRoutes);
 
 // Error handler
 app.use(handleErrors);
 
-const connection = createConnection();
+// Add exception handler
+createConnection().catch(err => console.log(err));
```

Comment:
Add also `then` handler here and start the application only when database connection is succesful